airbyte-workers/src/main/resources/application.yml

```diff
@@ -56,6 +56,7 @@ airbyte:
       image: ${CONTAINER_ORCHESTRATOR_IMAGE:}
       secret-mount-path: ${CONTAINER_ORCHESTRATOR_SECRET_MOUNT_PATH:}
       secret-name: ${CONTAINER_ORCHESTRATOR_SECRET_NAME:}
+      service-account: ${CONTAINER_ORCHESTRATOR_SERVICE_ACCOUNT:`airbyte-admin`}
```

Comment:
@andmosourceability You don't need the back ticks around `airbyte-admin`, unless you literally want those characters in the default value set for the property if the `CONTAINER_ORCHESTRATOR_SERVICE_ACCOUNT` environment variable is unset.

Put another way, I believe that this should be:

`service-account: ${CONTAINER_ORCHESTRATOR_SERVICE_ACCOUNT:airbyte-admin}`